Bottom: d557cbd52e0d957cca09a699ac5cce3f41a29114
Top:    a1a431acdfc15924ce55dd684833711b254d66e1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-29 13:32:32 +0000

Debug changes


---

diff --git a/src/hid/common/object3d_gl.c b/src/hid/common/object3d_gl.c
index 08625d1..16224ef 100644
--- a/src/hid/common/object3d_gl.c
+++ b/src/hid/common/object3d_gl.c
@@ -115,6 +115,7 @@ object3d_test_init (void)
 
   object_default_edge_appearance = make_appearance();
   appearance_set_color (object_default_edge_appearance, 0.0f, 0.0f, 0.0f); /* 1.0f */
+//  appearance_set_color (object_default_edge_appearance, 1.0f, 0.0f, 1.0f); /* 1.0f */
   appearance_set_alpha (object_default_edge_appearance, 1.0f);
 
   object_debug_edge_appearance = make_appearance();
@@ -287,11 +288,18 @@ draw_face (face3d *face, void *data)
 
   /* Object inherited appearances? */
 
+//  glDisable (GL_LIGHTING);
+
   glUseProgram (0);
 
   glMaterialfv (GL_FRONT_AND_BACK, GL_SPECULAR, white);
   glMaterialfv (GL_FRONT_AND_BACK, GL_SHININESS, shininess);
 
+  face_no++;
+
+//  if (!face->is_debug)
+//    return;
+
   face3d_fill (info->gc, face, info->selected);
 //  face3d_fill (info->gc, face, (face_no == debug_integer));
 
@@ -304,7 +312,6 @@ draw_face (face3d *face, void *data)
 //  printf ("Drawing face\n");
 //  g_list_foreach (face->contours, (GFunc)draw_contour, info);
 
-  face_no++;
 }
 
 void
diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 200f126..1c3182f 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2830,7 +2830,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
     glPopMatrix ();
   }
 
-//  glDisable (GL_DEPTH_TEST); /* TEST */
+  glDisable (GL_DEPTH_TEST); /* TEST */
 //  glDepthMask (FALSE); /* TEST */
 
   if (!global_view_2d)
